aggie-coding-club/Rev-Registration
# Logout button

## Description
This branch adds a working logout button to the website
## Rationale
Seeing as how it was possible to sign in, signing out seemed like the logical next step

## How to test
Login to the website. You should be logged in and your name and a logout button should appear where the login button used to be. Logout of the website, the login button should reappear and you should be logged out.

## Screenshots

|Before|After|
|--|--|
|Before Screenshot|After Screenshot|
|![LoggedOut](https://user-images.githubusercontent.com/16074716/92178559-6df2d000-ee08-11ea-8d32-d80bad42083d.JPG)|![LogoutIconOnlyArrow](https://user-images.githubusercontent.com/16074716/92334591-2a40d600-f055-11ea-894e-2303aa270daa.JPG)|


## Related tasks

Closes #329 #313
